models/staging/graph/graph.yml

```diff
@@ -6,7 +6,12 @@ macros:
     arguments:
       - name: column_names
```

Comment:
I believe this argument name is now `columns`

Reply:
you're right, corrected